python/fichas/dicionarios.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+"""
+# Ficha x: Questões sobre Dicionários
+
+Assim como as listas, os Dicionários são usados para aramzenar dados em uma única variável.
+Entretanto, a sua diferença se encontra principalmente nos índices e nos seus métodos:
+Uma lista possui int's como índice, enquanto um dicionário possui quase qualquer tipo de dado.
+Os dicionários possuem o que chamamos de chaves (keys) e valores (values) para fazer a associação.
+"""
+
+def traducoes(d:dict)-> list:
+    """
+    Imagine um dicionário em que há a tradução literal das palavras usadas como chave nos valores.
```

Comment:
Try to write these comments as a description of a function, like if it was the documentation itself. When you read it should be obvious what the function does.

Also, when you do it like that, the markdown file generated starts to make sense:

`Apresente uma definição da função traducoes(d: dict) -> list: que {DOCSTRING}`

Where the function docstring is inserted on `{DOCSTRING`.